### ISSUE TITLE
Using organizations in the name breaks node-lambda package

### ISSUE THREAD

**Issue by mboudreau** [motdotla/node-lambda#163]
If your package.json file has a name that uses organization namespace (ie. @company/name), it will break while trying to create the temp directory because the parent (the organization) doesn't exist in the filesystem.

```
=> Creating package directory
/home/mboudreau/Work/search-lambda/node_modules/node-lambda/lib/main.js:264
        throw err;
        ^

Error: ENOENT: no such file or directory, mkdir '/tmp/@company/name-1477450413533'
    at Error (native)
```

**Comment by mboudreau** [comment]
This is because of line 21 of node-lambda bin, where the default is to use the package json name:

```
var AWS_FUNCTION_NAME = process.env.AWS_FUNCTION_NAME || packageJsonName;
```

**Comment by DeviaVir** [comment]
Give the newly released `0.8.11` a try!
